### PR TITLE
CAEDD locale updates

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Adds CAEDD locale info. Removes dated EA stuff.
- **Is this PR related to a Monolith release?** Yes. 2024.02.0

### Resolves:

* [OKTA-685922](https://oktainc.atlassian.net/browse/OKTA-685922)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md](https://github.com/okta/okta-developer-docs/blob/a319ba4fa16830ce3e6ca5aac63e5ab14a04494b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-email%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4e4cfdff-9135-4077-a7f3-eb2d47e5f379) | 72  | 92  | 98  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.0f2eb17c-a6c0-4034-bd74-5e95e665663e?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by barbaravo-okta — APPROVED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md](https://github.com/okta/okta-developer-docs/blob/d49d01d1c4345133bd1dbe1b8d2deb223ce95447/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-email%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b3d00cc0-5397-4805-ad95-0cab5ac19a26) | 72  | 92  | 98  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.746c486a-b8b0-4e6a-8963-838e4d9a1331?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md](https://github.com/okta/okta-developer-docs/blob/2cb3c419c6e61b2f1ffde61b3b35226b65774436/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-email%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/992c7451-9ee5-4e82-af80-8181c7a417b4) | 72  | 92  | 98  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.32bfdb84-4dc8-4389-aef5-aee2426939a9?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md](https://github.com/okta/okta-developer-docs/blob/d2268a25589da09590dcedb59a314d7131d2d0f4/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-email%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/1bf42bc3-15a6-41b3-b4ef-d2478e2d081b) | 72  | 90  | 98  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.b11975c6-0bdb-4c76-8246-5e467040bcb2?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md [hunk 1]
```diff
@@ -186,11 +186,29 @@ After you customize a template in one language, you need to customize templates
...
+You can only create customizations using these additional languages by calling the Brands API. Once created, the new locale appears in the Admin Console along with the default-supported locales. See [Create an Email Customization](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Customization/#tag/Customization/operation/createEmailCustomization).
+
+Include `null` in the subject or body of the email customization. Okta replaces `null` with a default value based on the following order of priority:
+
+- An existing default email customization, if one exists
+- Okta-provided translated content for the specified language, if one exists
+- Okta-provided translated content for the brand locale, if it's set
+- Okta-provided content in English
+
+If you use functions while adding more locales, use the `getTimeDiffHoursNow` function instead of the `formatTimeDiffHoursNowInUserLocale` function. `getTimeDiffHoursNow` returns only the time value in the specified unit. See [Use functions for email templates](#use-functions-for-email-templates).
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md:204]: @barbaravo-okta This is new content.

**Review by barbaravo-okta — APPROVED** [review]

**Pull request merged**